setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -123,4 +124,31 @@ public function validate(InputInterface $input)
 
         return $errors;
     }
+
+    /**
+     * Get admin user data
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void
+     */
+    protected function setMissingValues(InputInterface &$input, OutputInterface $output)
+    {
+        if (!method_exists($input, 'promptForOption')) {
```

Comment:
Please let me know does this makes sense to you. I don't want to add too much wok :)